cmd/kops/create.go

```diff
@@ -147,6 +150,23 @@ func RunCreate(f *util.Factory, out io.Writer, c *CreateOptions) error {
 		}
 
 	}
-
+	{
+		var sb bytes.Buffer
+		fmt.Fprintf(&sb, "\n")
+		fmt.Fprintf(&sb, "Your input has been successfully parsed.\n")
```

Comment:
I like the idea of keeping the running list of objects. i'll write that up and get it back at you. Thanks for the good ideas.